.gitlab-ci.yml

```diff
@@ -0,0 +1,32 @@
+stages:
+  - build
+  - deploy
+
+build:
+  tags: ["runner:docker", "size:large"]
+  stage: build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker:v3892323-786b6fd-20.10.3
+  script:
+    - docker version
+    - docker buildx install
```

Comment:
`docker buildx install` was necessary to make this build use latest build improvements on the CI. 

On my local macbook the build was working out of the box... hmm Maybe I did that step in the past, and don't remember